Sources/MapLibreSwiftUI/Models/MapCamera/CameraState.swift

```diff
@@ -44,10 +44,32 @@ extension CameraState: CustomDebugStringConvertible {
             return "CameraState.trackingUserLocationWithHeading"
         case .trackingUserLocationWithCourse:
             return "CameraState.trackingUserLocationWithCourse"
-        case .rect(northeast: let northeast, southwest: let southwest):
-            return "CameraState.rect(northeast: \(northeast), southwest: \(southwest))"
+		case .rect(boundingBox: let boundingBox, _):
```

Comment:
I've created a `swiftformat` ticket  https://github.com/stadiamaps/maplibre-swiftui-dsl-playground/pull/27. We should wait for that to merge then we can let it sort out all the space/tab consistency.